checkov/main.py

```diff
@@ -112,6 +114,11 @@ def run(banner=checkov_banner, argv=sys.argv[1:]):
             bc_integration.persist_scan_results(scan_reports)
             url = bc_integration.commit_repository(args.branch)
         runner_registry.print_reports(scan_reports, args, url)
+    elif args.docker_image:
+        if args.bc_api_key is None:
+            parser.error("--bc-api-key argument is required when using --docker-image")
+            return
+        image_scanner.scan(args.docker_image)
```

Comment:
@elaygl This means it can't run side by side with the -d / -d flags... Why?
Can't we scan terraform + docker image in one flow?

Reply:
I believe it might be not very clear to allow both on the scan run.
What happens if in the provided `-d` directory there are Dockerfiles, is that clear that we don't scan their images?
For that reason, IMO, it's better to separate. WDYT?